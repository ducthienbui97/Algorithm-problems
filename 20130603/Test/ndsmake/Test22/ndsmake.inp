2074
-847579867 16644252 -593664701 -739151515 -964307442 -133829988 -562790198 760704196 154880773 455058070 -592016334 785019565 267648831 307755926 604210384 461326666 418663277 826135090 -854734926 667931418 -291196435 -424828062 -501162424 -786656443 677559661 61118403 -39289323 -614095132 -934565592 114501128 845496795 -835623424 -241807999 585396671 700156228 453227154 -179157068 -466573308 -169545337 292539307 244528470 649560962 119034339 -291735168 884295302 -99559579 -462150654 -702689136 292693707 -772964931 368648953 880900664 214127112 854700570 17122863 -596506188 853764812 -382717802 -409185664 892823841 736576280 -955013827 303331618 563149112 387101362 351020130 -742998731 921634712 841285346 390885850 -747695653 -721221553 187559386 132940066 515077297 140998609 -31430976 -31943264 -257427991 943106566 480954458 -731716350 -990058490 419313786 -956972971 -880165437 912134450 -748297800 149753138 -798862388 -660776897 37473200 -431868722 -55507144 126171989 -300951156 -560625362 398810532 -774769852 -830146527 -656057614 950193239 -343831650 870971387 773149432 -484059916 -267569930 289806415 36888622 886929510 982148029 -497552549 -467520342 853190332 -116394845 268471972 -572385013 -52442926 -435555333 -632124630 -830453190 815989499 -806855439 -200223023 -928809575 -567333114 939887644 416964434 761005260 -208250662 683918001 -746933947 -889109095 -674417913 -893860069 122752056 -710830331 482739597 631376367 943014000 -33081829 418318886 383243128 -666629508 -174554011 -321257696 773534284 922499591 -762308645 788362068 -474009020 312981062 -926282889 -55309073 633620759 -91023478 503418452 768227919 611299989 970196981 -616563838 -899494141 -477489538 -386941623 -875994007 532754165 -158567939 -281674276 -544224860 20438724 943342248 925847463 950257545 -486247995 -308565889 980537103 -198513765 -65205366 845513307 -1844345 482265838 -639527887 786267859 -64841134 371760246 80133384 -961449302 -188121942 293732801 -549062014 21693580 588993220 -983415982 84538981 -719260053 -315065308 183261624 -795757996 -469925995 -561976683 930065319 104068836 -222791801 -863568361 -733859206 393882968 933544146 301271986 814379676 532770262 -376313880 616918390 -181580726 -878111810 679676542 -694671043 -490096877 -361385688 302255159 -352374566 178396864 -400346505 -235938197 61135610 787442518 271951261 -249952380 789471349 768004529 -230200551 258152611 -490961879 967319642 -302447377 -690899935 -994510260 -386734481 -348397809 -559583998 -565168378 -223514936 -68488487 511845710 -228856242 950299786 -756388544 820742637 -310649055 19090841 -381080918 -784476066 887602602 316532486 -580071334 -687148735 -231923274 486723193 -45296874 886411130 425965962 -965189785 -843165021 -480021271 300940731 -288644853 681005811 373862137 993392212 -69280059 -874782314 940868280 310223469 447836775 490342348 365284326 -413003621 595026941 949110314 -389855100 189039512 419232191 30831433 26614690 -745313241 141396816 -778016890 744239426 606920942 719500470 750424481 118521780 579502026 360865400 676207727 -669885983 696644437 -470392334 192202031 258169908 -951513465 97769817 478708986 -723059162 773666124 -253288615 995455589 320619075 -849227246 703514609 -792453215 280363271 36660854 -69586599 88120307 278521351 204817980 -168462580 -90196758 -131809551 708776511 613053562 753970738 817014844 967577640 -621135047 832405731 91126698 551683021 551713368 650347876 -999136019 823002147 856213716 -111373830 -487465370 66282978 758310180 906442737 405239816 -399477324 -600695825 549775963 182606873 -799463458 -46960211 46798749 210576602 -344523637 554551909 -670057213 -29195225 624621335 -560155107 181506895 572993925 -718744489 -806133988 368381625 -942518122 639528091 546030025 413987803 -65282796 684958369 763715383 120102332 -628593394 121197878 35417066 -348715453 123996409 177566266 -146417445 808423557 -416318324 194913277 -600893880 -69833560 281210119 32014467 -656105855 -349204330 -415074928 -68736191 780267039 704663869 138209215 -691070860 773033300 -653982080 159306293 925310046 831308733 -714862745 781389471 898790678 168021755 -946585991 -988270356 70914981 110502064 -616912827 745511534 -642770859 -31599059 -988794246 739523193 -293055355 447094951 -149544223 -878373017 -246221641 -729163037 -339045189 738809169 -663324221 943558525 -687103417 -280252822 -944057553 -70821420 357488375 291671190 -698111083 747571778 -103160082 -297000892 293380323 -3576979 185828139 618963172 -292934368 -871524958 670310473 349572176 877398619 434501898 539187491 141474184 -72157866 980218998 -407244057 -505121857 -799186894 -979262527 -786315053 149746830 -630954567 -42470830 -144279152 97423179 -369596600 -182447170 -547532761 220659573 -979804329 139684934 405639847 -517417714 -285114298 952357739 628746759 -643293068 -130569889 431303888 -170446368 -15963441 921745812 -783711320 801094058 -519350505 -43773924 -663563081 -653060922 -653494500 -221472956 -963111065 -32520123 -541545349 -177585489 -706234993 -940838156 -229924989 -958889409 744660215 -605423838 -641969453 -353798496 839827072 -408086895 869178623 860166689 -778750652 -384089127 478539542 -927997792 -94356754 299495058 -719025056 -663412213 -580562860 -718989657 677526510 20745087 41700519 4633081 766842499 -127118401 -265142178 215102813 -613140825 822052574 -841018311 -737268632 -163029777 75817616 794848982 -161405805 747359463 398126689 -892591773 369746184 -8923638 367205772 741829998 -448085749 526035694 575599737 921933273 310698760 -841463683 -358626345 469060841 868101685 -412922019 618752849 -509365891 -682226461 -943271567 -592357905 -81482405 467616103 -622316167 497566220 -7887194 -103686241 452489941 688690796 708708199 -284268023 -380324260 -656340470 -862639723 120067531 -374663276 -574897066 299489390 723002401 166106265 -332883971 382176228 609576590 969650272 -279687787 -812357833 25933285 -278129527 30460314 -892735629 -504725333 933076575 -642029552 771983074 -429581110 286757520 -426109001 121302789 968179941 -981088673 816437028 -491870754 8110419 -375624666 -120575084 387646031 489837225 566252228 539276826 -353513882 865580666 -984179130 -251939011 -869871366 -189366880 844252303 -314476913 269702034 -577607444 -972404888 -827817134 637565691 639796531 -567194647 810167086 70937721 234816484 847631302 675547516 504563522 -81037261 285500185 498612055 -408914144 -952090717 -161828934 -919951980 406943544 -933965514 338684450 296728849 874852355 91963035 -639209795 561457828 538151508 -75964353 -51627968 -357022249 508579924 -354652178 360183065 967763668 -400385682 119329792 -237041503 25085587 -852665939 919979237 -108891811 -931454874 -273029495 61908456 515024921 880358292 -96993662 -119132592 -27657087 -272468638 -222164287 -242379004 438663992 771376809 120977584 814527863 639022234 260599572 834951636 -963265148 829510836 -152797204 966650579 867159960 -982198461 -586943696 152046159 -345595184 436657190 -750805199 -763110481 530463064 -229207202 960615906 -157488133 -405105627 771651299 -821875406 423271477 -485402306 227351006 -363610566 529707572 -163017890 42779481 -540755042 -686134179 -308164218 265407500 -481634559 -108408969 421992361 462310114 -456204973 -248663596 419082476 -858312700 305857537 926179355 -303528118 195755442 654527562 -298580040 982907491 706454344 -879002692 -433587420 680109982 96955506 632037043 -587602694 -273106720 -128370261 -138276695 -723560001 -349535479 449025251 47698337 -217379367 -555206680 404563930 -54197880 -995370113 -694340071 899594963 136419241 643129024 -762589099 750964508 -348786888 -666801728 548157689 -579220764 14617914 -183402639 125727848 228557240 -223389576 102273429 124075852 -395933034 -722100593 159274353 779736827 736402863 -617736155 171637008 -668675138 984235574 785163056 -547031413 -833076992 -940088692 -806041650 -169742950 -980744753 785980749 -345900215 761343860 743149279 -877936210 395021877 -568610593 898850579 -254530618 -523314241 513563901 -22718083 414373048 686039725 12795377 -184983389 901050688 -976795452 180515869 -859090160 -636283792 -146716122 842123833 947552867 -5294951 48059939 10080130 -278756269 26183181 598369905 138937067 989182924 -681628390 55728857 901948957 210416518 662587353 -784874750 -462631165 -242114474 68507767 955709434 334677113 613227224 771057080 -924316031 592341493 -919043780 -201896853 576750204 654237982 42895535 500300976 963821605 -637229772 604473466 668815250 981016208 19196202 419678882 -136751463 185068760 -662376830 878089762 -540421508 -166008853 -49239981 -918199606 -466803981 512230127 -182367124 -936365900 515449259 -382333872 -750742381 -584213154 29194835 -709600920 -542141978 448065641 -781045293 809059017 -979102621 261749229 -789434723 375380778 371050627 -455480597 -203141888 -71795552 -174726328 -296625594 886153912 -96767925 688747099 998822932 2128042 -105519238 -76288611 -587723975 -567544291 232417485 236581724 -478106450 -66665249 -855516263 -432407304 -291198935 648625579 621574561 -849058022 872094059 -900548444 -100801932 -162624763 800439334 -919067650 364475528 953819836 512515868 829232131 211981928 -511765101 -374531097 955561530 -257018612 326209518 -419883260 -158571115 132766475 -147982976 -600736306 842268168 -804173502 353951274 445577977 -310887578 898054712 -78913277 952359492 -993456405 615429581 -77158839 -781572590 -83882607 -409467557 537337664 780226346 674890318 -438266254 82437688 -62540979 -416003106 -364437690 946396035 -112079257 -1579732 -951486027 -950609189 -511285343 -393551035 319205201 407361440 -570825350 134127727 997293067 -953205955 -829781631 -969652653 392171901 -960329816 716598700 -256210603 -657947437 246177799 -126894760 -792557820 969273687 -419677840 -229555234 697751981 -143066727 -327084599 712968910 165614346 68612407 855755470 -566206005 272298008 254122318 262895409 -610514429 400799750 -459933315 -139008036 -73998519 -516466615 667910034 817756368 326627276 -681234476 659870003 432249669 -71217171 -206208034 -837650422 7566499 456169073 -534533583 444812876 -825370717 -419657457 135770304 -899211902 945115987 -248033485 -95895233 -691199539 -213516267 -950676558 160047482 -397606316 -562913406 -149376037 964948251 15182909 915309508 -186422326 -625978502 -637474764 -856596911 509921610 -328640776 -182601792 -736843827 789603953 -500713476 88208817 531586205 -225842074 316833859 -151523977 253669131 -241485811 -622011371 -81861035 -404048305 -507784259 225621504 -279876390 701250163 -743575982 148780928 113104079 -44879659 907306698 -301339569 391058534 485065809 -194239542 200836905 -286884252 874404097 978459507 -539970098 841399540 -785814875 -74868888 -1530948 470263668 815186757 960052584 -682632230 -900241703 -839441246 711080866 398533626 801539521 -505436935 461130408 -977784656 707286811 2268290 22960375 -599328651 -252765652 -356674680 630200491 953510648 688715367 -248159141 -280776467 35514667 -483975686 -794420545 761929913 59518526 -968817119 -438374783 -145985443 -728382162 -320444702 951790010 823497161 175499904 288771223 -560776024 758026038 -556372955 564141902 423471473 48914395 451541087 682769134 128597725 -668957526 748545834 971284156 431333037 602544282 358340898 -91761926 763924372 844518431 890641170 649152170 793785558 -171534916 549615642 -418347152 898884988 -330227282 431473788 -861088363 834600241 822799049 -232977661 155230397 629921377 241035024 -647936705 92868635 -192541345 -888785598 -512152997 -137641928 838863987 706733249 268120297 40310347 298468712 97538856 489596645 12992203 71582611 -467714803 364364697 -450938350 160589157 -668472019 -744458109 959895692 -349938400 -669649863 905314459 -738342234 936989495 16191510 280442891 -323657619 991077015 -660494746 979962669 -280779554 596741010 -402290946 695151553 -795284586 -897007514 -735966763 284810944 42173030 -911634231 732896733 37246577 937811327 270485182 119683017 379357312 -395595259 109600886 -124602152 257126023 -576196724 661653348 99252945 -636409 969302887 996532127 390831533 -664074072 -285100869 716290802 -468848058 995584669 -511784898 358622621 501017894 718824849 -698039067 952374984 -613311697 708417592 639764231 141189494 646579734 661112608 109758602 316544569 900045684 209699023 -516156018 -274341992 -398800633 980034436 -2099811 -792560220 -292390854 -592551609 -164367384 722583768 938177034 -485650225 238099899 964745105 -313663124 60622282 577328637 128428130 -539646053 -809114677 143597270 706295829 218391440 -151341618 -31122581 243938623 512704144 -910511000 -56316987 816210815 -854872097 957023375 -54582065 855588947 -358643310 -58140413 802375294 -667156391 57192971 977763738 187597149 -234878558 782058076 412374238 -670022442 -715994692 -385070975 433128812 567850236 641313588 871747161 307205074 492557015 158081365 356107706 175702545 -419052844 -332396716 -919686616 204632442 -584333681 -775413842 -556198646 877334788 -177860019 369740818 547274940 14555094 -848015628 -300559597 -648433299 228021492 22101667 988419449 16359955 845035750 908907330 62699450 581709916 -87337150 684621460 514357893 703915191 305048759 210973009 66674584 441979485 815256714 -8722935 839881537 937167758 184013861 841779571 616966971 -425865043 457414635 -30018773 -663499085 508139656 -301382823 -124887874 -385590602 210512055 -343410715 519627965 -769630694 -209304966 623867343 -116670962 760540548 -209606599 -17600105 -647530291 -682527626 438966602 -908406827 -416767512 914777603 -610380039 -156220080 638009626 318926803 -112547740 -864397615 -388140710 879270724 -321814026 611992595 -681650717 896278187 -694359426 -555651114 661525985 654885048 898889490 -337237835 -151334921 -284140122 734546249 100654309 988270707 -80533876 -46222347 -799497309 -329703051 397825159 323547937 564546296 322698602 167739336 -804718316 838070413 556684770 625618851 -363905241 310318870 -980013643 652986118 158601656 579889244 -109497536 -426803561 -846436041 -60523315 -562452886 613548638 227567889 -279246500 966756221 34315508 199935438 -480461581 -37627141 -856588647 -138690175 706856445 123621195 -676881128 -382465008 807880991 744004468 -747284869 744569380 -17981968 -870954224 573631020 -88584264 -392275610 645848375 -882213607 115626375 -413978997 50649302 233985481 633865262 788286832 121106669 371907641 -761164652 -763693612 -784594744 609706181 -996811442 261060640 -897308631 -73662688 68175892 -68004115 919893868 772080335 -566304201 -803814732 39466886 894228043 -567816973 452925778 408700917 -80944409 134516235 -103575104 765710486 -867810721 -771559960 -791427962 -830478671 581732902 530658694 821380185 -775547407 715700700 402661631 -68690210 -274186496 -473997604 537735346 733865869 -67402305 342105841 615031568 -172290384 -11687649 771095592 -208338685 -742210624 -256719109 -570140522 -972887198 671431230 -160869227 -749245210 187389164 741914269 -456725270 -491592278 -286841991 399900323 -176934542 -774876976 -641789958 750278223 263463316 -362196537 158570413 -131672977 -303019829 -484899488 -922120361 32321101 375481718 859603406 406482977 -516928375 81752636 -491514676 31873037 870193922 157850649 127861213 -221554669 -958585362 337465098 751149743 69552604 -912804342 -51895471 528608153 -225245414 -263766549 -982837914 431257155 -330204030 547359346 -474475910 466900069 673886823 -246790051 40993704 780799144 805209332 -585854725 -942661189 -961920088 808216070 -669167624 55966614 -404049125 604054180 846427222 105593156 -495701749 742475291 772013692 -65026659 127838527 -946318150 -466543256 907389073 -691008840 -423708633 -615994111 -741704684 -221951145 -886319267 -722270810 -241384548 -204456916 -293429634 -118799223 417663209 -39265092 344292961 -356993929 -416723771 -899888868 -922273992 -780346881 -759556754 737723395 -491649136 680767084 450636417 -164833094 -477342501 -272409059 947026282 339752758 15565546 601778737 -24475474 472585824 659189973 -870312181 -75841969 379575381 150094686 -807403393 -333621721 -712280181 350935382 -919977040 292129574 850200264 -635340277 -952008589 739158174 654258572 -780510550 -67008379 -652103102 965155308 1928004 271541139 -382463731 290910058 -911380460 939351770 702859434 838291058 346622190 337619294 102128541 379284639 145991010 -713358415 258868089 -307381535 769674406 -1513683 479101068 579616841 699253137 -674978098 246084280 -378300556 -75914044 49513385 -791335583 308025490 -441952541 -184594675 -923789182 -67413486 -946242303 -243608523 -689581862 -388705812 584919322 434168742 600463684 -270073462 303810745 -401442261 -619786459 229586515 300096442 170270059 -118082561 -409696042 -47409878 -285071324 -633602526 -605330288 -451565494 -287481850 846054017 508396123 -217537613 -891498706 -67343966 261545843 245161107 -784630979 -476151377 286710565 786369557 257438741 -547070038 -468892394 899604703 569893310 955049653 -989205348 -284375888 -396317154 994313787 -980846953 337878644 928960404 -267649321 38716611 932229084 943109370 878075061 -39023098 -396828821 860100583 -384987932 994151424 -418754744 -469293788 744870301 425445162 526065742 -711477414 187700288 -464956835 -612772408 236722442 -623517414 -897103601 -529620092 240049797 -690804861 -138602510 796494792 -700615096 585506398 -988438962 301120209 60686686 782247295 -979599365 647200308 532735584 -540896902 813144524 -130280965 527283444 -131915112 153293464 -133308954 670074598 294893795 695808742 188548307 -495052575 623900513 -327693004 -318224328 -520194871 564963421 432513252 905208268 573679544 955241696 229032388 316151109 -190866028 -42622657 -845304264 580410429 -545530806 183836878 -252223994 482881070 75068082 250557374 -69892452 -695779619 -673801772 -527672636 -618304686 80324742 807838194 936426492 -560278332 -812131724 -371567856 564165267 914785281 -931639711 -838097360 -447800820 404528141 -836631673 -819049064 493324173 29566620 -606838137 894490685 -199881254 -742706800 -441446395 -772755851 190148375 -665453710 -517873557 974329999 364961090 -224576868 248230942 -15481858 -248845616 -587272548 -693107562 -479205808 -684569755 -142757569 -778945209 959114407 -154846152 -969599149 -732593838 -501203379 723664689 -133291614 -625219255 258588665 338479213 -253393418 916523106 242526593 -60676339 36718877 -968537256 457753772 -382209894 694728460 721038047 833324838 -143296236 423066400 876923842 -357721715 -127368810 -293160321 -250853734 -850887256 -93460569 -462867811 -901615316 -132438601 432652063 924679643 -599153824 53440229 -891887437 127254602 -803237670 315397920 804134540 -281216969 -871254931 -245779216 -767524645 37448934 -755956873 364838124 -597015516 84826272 -919690302 -360224993 -457139381 -104748155 239742753 662340242 -985626155 -53140663 249775892 980404807 -913467959 928433259 961265674 410541572 867736177 986606165 212258498 472461267 -883244115 -312786923 779066286 -363601757 305892019 -358085254 -252709861 727212885 957316775 701608284 -391848015 -479692759 258124157 661922622 644071880 -244944137 483651994 -706281495 498516928 -808699643 -929051349 -211369282 -780015382 781252518 467439906 120521257 470311678 -775785009 -834748106 -534471238 190308064 111217431 834603070 -591469125 -784198696 -599427501 857020089 -160984101 72323652 -852462649 128753719 411846592 -874297955 883996093 -174348694 625787293 884185556 275110717 562602587 -780543892 -153825603 961758770 -727561601 672630111 -773917218 268308178 -44510886 211075514 262837769 -904830234 -363779895 514325162 85780362 -424654882 279272281 809346362 -694713375 -699404139 68946132 411619992 -571109826 891353483 720525912 -968393945 497669684 710073875 322906292 265566971 -104732436 -272882531 289722816 -851592863 -583728974 -961701390 -499263493 -680055217 767743188 -291589109 674577362 80574452 891390552 -483643113 268384031 -49922160 838478875 422100827 286691583 629364191 -977911827 -361101149 -746447678 387918880 -111785000 -167724189 -305793541 913544128 944450486 149196456 375408040 582009940 -94262071 274438688 -845768246 -532764847 -634689725 163348062 427372006 722366197 -243487486 -304614826 489492321 -838085055 -644156944 -959217173 738035083 288678894 183777287 354927865 618341465 -779766168 731682547 -329384600 983873655 -780459877 -543361497 250757370 795000176 -752704477 294589149 14329374 -636873616 94212730 -124285441 218485996 -618369746 -66542387 -400373905 432272602 312084274 -675291632 51793091 -32187233 -448824728 459569303 -414391747 146494933 -400718975 -48495650 -376933090 289621274 2546 664218765 -174750518 -418313884 468755623 -153974112 347769088 770727974 -702620590 -50463455 -563696923 -632811739 5818059 125055945 -22494024 831200934 -493978891 -333261966 -614370824 561799945 -801193452 -471892295 -330898817 -527688992 842826398 134341027 748127711 -930891795 -495507937 769720884 -84391929 -473389581 -733510044 -924227238 -502836814 309074013 330065461 -907729706 192222474 780597051 432993222 -978207291 708199014 71131609 -3162041 327863755 -788461772 535396606 -648388970 647003355 130816242 -84999490 82770709 -316943964 816590422 271858172 104095381 124214939 173320334 -908887168 -199627151 -458132063 -557166942 837218993 -935830600 -890917806 866327797 -637876241 -539438206 -475129110 -996390166 55906959 611689059 388587475 427804834 -880758668 243555582 -553053745 -278620444 -872432314 822656945 -480509102 804296558 -242641651 -666914456 -459841538 -643029591 -923884101 -195862768 -303843865 318476653 -273814263 104441519 539600537 161552266 -388044866 879255763 803139050 951110633 -543442156 -499541354 413194483 582082048 654122083 93647532 -942509472 -837391193 243030744 -530997213 3031915 215054224
