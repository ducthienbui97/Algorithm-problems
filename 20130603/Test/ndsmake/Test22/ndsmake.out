1029877123351
